app/Http/Controllers/UserController.php

```diff
@@ -74,9 +85,9 @@ public function store(Request $request)
                 $user->assignRole($requestedRole);
             }
         }
-
+        
         if (is_numeric($user->id)) {
-            $dbUser = User::findOrFail($user->id);
+            $dbUser = User::findOrFail($user->id)->makeVisible('api_token');
```

Comment:
Why are you doing this here? It's made visible in line 117 after permission checks.

Reply:
Those are two separate functions?

Reply:
this is a good point